storage/wdpost_run.go

```diff
@@ -150,6 +150,15 @@ func (s *WindowPoStScheduler) checkNextRecoveries(ctx context.Context, deadline
 	if err != nil {
 		return xerrors.Errorf("checking unrecovered sectors: %w", err)
 	}
+	
+	// check some unrecovered sectors can not recovered (such as cache file is missing) 
+	sbfCount, err := sbf.Count()
+	if err != nil {
+		log.Errorf(" sbf.Count() | %v", err)
+	} else if sbfCount == 0 {
+	 	log.Warnf("unrecovered exists sbf.Count() sectors can not recovery, exit this function.", err)
+	 return nil
+	}
```

Comment:
`Count` failing here should just return an error - if it fails, sending the message will fail anyways

```suggestion
		return xerrors.Errorf("counting recovered sectors: %w", err)
	}
	
	if sbfCount == 0 {
	 	log.Warnw("No recoveries to declare", "deadline", deadline, "faulty", uc)
	 	return nil
	}
```